content/en/docs/Architecture/meta.md

```diff
@@ -10,33 +10,42 @@ We like to define Kairos as a meta-Linux Distribution, as its goal is to convert
 
 ## Kairos
 
-The Kairos stack is composed of the following:
+The Kairos is a software stack is composed of the following:
 
 - A core OS image release for each flavor in ISO, qcow2, and other similar formats (see [the list of supported distributions]({{< relref "../reference/image_matrix" >}})) provided for user convenience
-- A release with K3s embedded.
-- A set of Kubernetes Native API components (CRDs) to install into the control-plane node, to manage deployment, artifacts creation, and lifecycle (WIP).
-- A set of Kubernetes Native API components (CRDs) to install into the target nodes to manage and control the node after deployment (WIP).
-- An agent installed into the nodes to be compliant with Kubernetes Native API components mentioned above.
+- A release with K3s embedded (optional).
+- An agent installed into the nodes to manage the node lifecycle.
 
 Every component is extensible and modular such as it can be customized and replaced in the stack and built off either locally or with Kubernetes.
 
+### Requirements
+
+In order to convert a Linux Distribution to Kairos, the distribution must meet the following requirements:
+
+- Either use `systemd` or another init system
```

Comment:
correct, technically we are not tied to those projects but we adopt them and as such we do require it. Nevertheless an user could create their alpine version from the same framework images by just adding the necessary pieces. I've added a stricter requirements in the "From scratch section" and left it loosely ended here to avoid to be too much technical, but let me know if should rephrase it

Reply:
gotcha